### ISSUE TITLE
Remove the title attribute breaks Genesis Simple Share

### ISSUE THREAD

**Issue by robincornett** [RRWD/genesis-accessible#16]
The "Remove title attribute from links" setting in GA breaks the output of Genesis Simple Share, because the GSS divs have a data-title attribute which is stripped out by this setting.

Example of the default GSS output:
```
<div class="googlePlus sharrre" id="googleplus-after-7573" data-url="http://local.wordpress-trunk.dev/a-gistpress-example/" data-urlalt="http://local.wordpress-trunk.dev/?p=7573" data-text="Hello World" data-title="Share"><div class="box"><a class="count" href="#">0</a><a class="share" href="#">Share</a></div></div>
```

The stripped out version:
```
<div class="googlePlus sharrre" id="googleplus-after-7573" data-url="http://local.wordpress-trunk.dev/a-gistpress-example/" data-urlalt="http://local.wordpress-trunk.dev/?p=7573" data-text="Hello World" data-=""><div class="box"><a class="count" href="#">0</a></div></div>
```

You can see that `data-title="Share"` is changed to `data-=""`, and the `<a class="share" href="#">Share</a>` is removed altogether (although this second behavior seems odd to me somehow). No idea how to fix, sorry.

**Comment by robincornett** [comment]
If you change [line 23 of attributes.php](https://github.com/RRWD/genesis-accessible/blob/master/includes/attributes.php#L23) to check for a space before the title in the link, it seems to resolve this, but I don't know if it would be a better fix:

`preg_match_all( '| title="[^"]*"|U', $text, $result );`

**Comment by patrickodacre** [comment]
Thanks for posting this fix. I came across this issue myself just yesterday.

**Comment by RRWD** [comment]
Thanks Robin, changed it in attibutes.php

**Comment by GaryJones** [comment]
Which commit?

**Comment by RRWD** [comment]
The one I will do this afternoon

**Comment by GaryJones** [comment]
If you commit, and add in "Fixes #16" in the commit message, then when you have committed and pushed, then this ticket will close itself, AND have a reference to the commit, for an easy audit trail.
